Kitodo/src/main/java/org/kitodo/production/forms/ProzesskopieForm.java

```diff
@@ -128,6 +139,8 @@
     private final String processListPath = MessageFormat.format(REDIRECT_PATH, "processes");
     private final String processFromTemplatePath = MessageFormat.format(REDIRECT_PATH, "processFromTemplate");
 
+    private final TitleRecordLinkTab titleRecordLinkTab = new TitleRecordLinkTab(this);
```

Comment:
That's true. This is not to increase the class `ProzesskopieForm` even further, it is already too long anyway. In the new implementation of the metadata editor, we have already implemented this in such a way that every panel and every dialogue has its own class.
The class is called _Tab_ because it represents a tab of the HTML page. So far, only classes that represent the entirety of an HTML page end up in _Form_.

Reply:
Please rename it to Form